source/_static/js/main.js

```diff
@@ -30,3 +30,35 @@ window.addEventListener("DOMContentLoaded", (event) => {
 		$('body').toggleClass(target + '-toggled');
 	});
 });
+
+window.addEventListener('hashchange', function(){
```

Comment:
@rushenn @kaankabalak this might be the hackiest JS I've ever used, but here we are.

I've integrated a new library https://github.com/executablebooks/sphinx-design which replaces some old contents with nice shiny new bootstrap/material design components.

The hack below allows tabs to "switch" when an anchor element is hidden (due to tab behavior).

It's kind of janky, and I have an open issue to maybe reduce the jank (https://github.com/executablebooks/sphinx-design/issues/30), but absent that - does this have any obvious "this might break a lot of things" consequences I haven't considered?